get-secretmanager-secrets/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "get-secretmanager-secrets",
+  "version": "0.1.0",
+  "description": "Get Secret Manager secrets GitHub action",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "ncc build src/main.ts",
+    "lint": "eslint . --ext .ts,.tsx",
```

Comment:
Please use prettier for this step to ensure formatting consistency with the rest of the repo.

Reply:
It does - this is the recommended settings from prettier. The eslintrc extends `plugin:prettier/recommended`, which applies the prettier settings.

Reply:
https://prettier.io/docs/en/integrating-with-linters.html#eslint

Reply:
Makes sense. In that case, would you mind updating the settings in the setup-gcloud action to be consistent?

Reply:
I think I'd prefer to do that in a followup PR. The diff is pretty large: https://github.com/sethvargo/github-actions/pull/2